### ISSUE TITLE
Ignore Material Information

### ISSUE THREAD

**Issue by lab57** [g4edge/pyg4ometry#111]
Hi,

I'm attempting to use this to convert a gdml file to .obj format. The material information is not necessary for my uses. I'm getting a number of errors related to parsing the materials, so I was wondering if it would be possible to just ignore them entirely.

Here's the most recent error I have been recieving

```
Traceback (most recent call last):
  File "/tmp/Physics/moller-geometry/./export.py", line 4, in <module>
    r = pyg4ometry.gdml.Reader("mollerMother.gdml", reduceNISTMaterialsToPredefined=True)
  File "/tmp/pyg4ometry/src/pyg4ometry/gdml/Reader.py", line 38, in __init__
    self.load()
  File "/tmp/pyg4ometry/src/pyg4ometry/gdml/Reader.py", line 116, in load
    self.parseStructure(xmldoc, materialSubstitutionNames)
  File "/tmp/pyg4ometry/src/pyg4ometry/gdml/Reader.py", line 1402, in parseStructure
    self.extractStructureNodeData(node, materialSubstitutionNames)
  File "/tmp/pyg4ometry/src/pyg4ometry/gdml/Reader.py", line 1447, in extractStructureNodeData
    self.parsePhysicalVolumeChildren(node,vol)
  File "/tmp/pyg4ometry/src/pyg4ometry/gdml/Reader.py", line 1480, in parsePhysicalVolumeChildren
    volref    = chNode.getElementsByTagName("volumeref")[0].attributes["ref"].value
IndexError: list index out of range


```

Thank you!

**Comment by stewartboogert** [comment]
Can definitely add a flag to skip the materials. Are you sure the materials are well formed in your GDML file? 

So I'll add a flag to skip materials and if you don't mind can you send me the offending GDML (in confidence) so I might see if I have a bug with reading the materials (Stewart.boogert@cockcroft.ac.uk)

**Comment by stewartboogert** [comment]
Fixed with #117
